csm/plugins/cortx/alert.py

```diff
@@ -176,7 +176,7 @@ def _plugin_callback(self, message):
                     """
                     Calling HA Decision Maker for Alerts.
                     """
-                    if self.decision_maker_service:
+                    if self.decision_maker_service and status:
                         self.decision_maker_service.decision_maker_callback(sensor_queue_msg)
             except ValidationError as ve:
```

Comment:
Acknowledge in ValidationError but not in other exceptions.

Reply:
Done